src/ts/client/grapesjs/index.ts

```diff
@@ -384,8 +399,8 @@ export async function initEditor(config: EditorConfig) {
 
       // Remove blocks and layers buttons from the properties
       // This is because in Silex they are on the left
-      views.buttons.remove('open-blocks')
-      views.buttons.remove('open-layers')
+      views.buttons.remove(cmdToggleBlocks)
+      views.buttons.remove(cmdToggleLayers)
```

Comment:
cmd* means "command" but here it's an ID your using it for

so maybe rename cmdToggleBlocks toggleBlocksId if it makes more sense?

Reply:
Will do that! Makes sense.

Reply:
Oh by the way, the reason I named them like that is because I've seen this convention at some other places in the code. I was in the process of maintaining the coherence.

Examples:
![image](https://github.com/silexlabs/Silex/assets/44942598/5c3c3d7f-d57b-4a94-9469-ff2fed6b07e7)
![image](https://github.com/silexlabs/Silex/assets/44942598/495a7b25-fa43-454c-b29a-d95b5ad2ced8)
Etc.

Let me know what is your verdict.

Reply:
ok let's keep it coherent then :+1: